### PR TITLE
XS-2890 DQC 0014

### PR DESCRIPTION
**Pull request by mindeerickson-wf**
##### Pull request for: [XS-2890](https://jira.webfilings.com/browse/XS-2890)
[XS-2890](https://jira.webfilings.com/browse/XS-2890) - DQC 0014

-
#### Changes:
- Created DQC Rule #14 
- Added the rule message for rule #14 to mapped_info.json
- Added dqc_14_concepts.csv
- Moved concept_map_from_csv() from dqc_us_0015.py to /util/neg_num.py
- Moved grab_numeric_facts() from dqc_us_0015.py to /util/facts.py

#### Testing Notes:
I have attached the filing documents (and associated project) to the Jira ticket that can be used for testing.  The rule should fire two times.  
-
Please review: @jasonleinbach-wf @chrislococo-wf @andrewperkins-wf @brianmyers-wf @bradbenjamin-wf @derekgengenbacher-wf

### CODE REVIEW

**Comment by mindeerickson-wf** [comment]
Original PR:  https://github.com/DataQualityCommittee/dqc_us_rules/pull/83

**Comment by brianmyers-wf** [comment]
+1

File: dqc_us_rules/util/neg_num.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import csv
+
+
+def concept_map_from_csv(neg_num_file):
+    """
+    Returns a map of {qname: id} of the concepts to test for the blacklist
+
+    :return: A map of {qname: id}.
```
Comment (chrislococo-wf) [inline dqc_us_rules/util/neg_num.py]: SG: Missing param block for neg_num_file

**Comment by brianmyers-wf** [comment]
+1 to docstring addition

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by chrislococo-wf** [comment]
#### QA Steps: 

- Manual testing: Ran a sample document through arelle app, 14 is firing like a champ.
- Flake8 on dqc_us_rules: 
```
(dqc)  chrislococo [dqc] (XS-2890) $ flake8 dqc_us_rules
dqc_us_rules/dqc_us_0014.py:28:80: E501 line too long (80 > 79 characters)
dqc_us_rules/dqc_us_0014.py:32:80: E501 line too long (92 > 79 characters)
dqc_us_rules/dqc_us_0014.py:48:80: E501 line too long (92 > 79 characters)
dqc_us_rules/dqc_us_0014.py:49:80: E501 line too long (93 > 79 characters)
dqc_us_rules/util/neg_num.py:8:80: E501 line too long (84 > 79 characters)
(dqc)  chrislococo [dqc] (XS-2890) $
```
- Flake8 on tests: 
```
(dqc)  chrislococo [dqc] (XS-2890) $ flake8 tests/
tests/unit_tests/test_dqc_us_0014.py:46:80: E501 line too long (93 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:56:80: E501 line too long (92 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:57:80: E501 line too long (92 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:58:80: E501 line too long (92 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:68:80: E501 line too long (111 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:74:80: E501 line too long (111 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:80:80: E501 line too long (114 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:86:80: E501 line too long (118 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:92:80: E501 line too long (111 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:98:80: E501 line too long (115 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:102:80: E501 line too long (80 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:106:80: E501 line too long (118 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:114:80: E501 line too long (111 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:120:80: E501 line too long (114 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:147:80: E501 line too long (94 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:148:80: E501 line too long (109 > 79 characters)
tests/unit_tests/test_dqc_us_0014.py:160:1: W391 blank line at end of file
(dqc)  chrislococo [dqc] (XS-2890) $
```
- Nosetest result: 
```
(dqc)  chrislococo [dqc] (XS-2890) $ nosetests tests/
........................................................................
----------------------------------------------------------------------
Ran 72 tests in 0.046s

OK
```

#### Result: QA +10, followup created for flake8 corrections

**Comment by andrewperkins-wf** [comment]
We have opened a followup ticket in our internal tracking system to clean up the Flake8 errors.

Merging.

**Pull request merged**